clients/admin-ui/src/home/HomeLayout.tsx

```diff
@@ -5,25 +5,33 @@ import { ReactNode } from "react";
 
 import Header from "~/features/common/Header";
 import { NavTopBar } from "~/features/common/nav/v2/NavTopBar";
+import NotificationBanner from "~/features/common/NotificationBanner";
 
 type HomeLayoutProps = {
   children: ReactNode;
   title: string;
 };
 
 const HomeLayout: React.FC<HomeLayoutProps> = ({ children, title }) => (
-  <div data-testid={title}>
+  <Flex data-testid={title} direction="column" height="100vh">
     <Head>
       <title>Fides Admin UI - {title}</title>
       <meta name="description" content="" />
       <link rel="icon" href="/favicon.ico" />
     </Head>
     <Header />
+    <NotificationBanner />
     <NavTopBar />
-    <Flex flexDirection="column" gap="40px" width="100vw">
+    <Flex
+      as="main"
+      flexDirection="column"
+      gap="40px"
+      width="100vw"
+      height="100%"
```

Comment:
copied over the heights from Layout. the difference was noticeable once you had the banner in

Reply:
Ooh, gross I didn't see that we use a different `Layout` here

Thanks for doing that. Is the `100vw` actually needed here though?

Reply:
(for additional detail, `100vw` isn't great CSS as it will fight with scrollbars. You can merge with this as-is, but just calling it out)

Reply:
oh good point, it looks like it's not doing anything. I'll remove it!